code/modules/antagonists/eldritch_cult/eldritch_items.dm

```diff
@@ -252,3 +257,31 @@
 	icon = 'icons/obj/eldritch.dmi'
 	icon_state = "eldrich_flask"
 	list_reagents = list(/datum/reagent/eldritch = 50)
+
+/obj/item/clothing/glasses/hud/toggle/eldritch_eye
+	name = "An ancient eye of a forgotten god"
+	desc = "Allows the user to swap between three hud types, science, medical, and diagnostic"
+	icon_state = "godeye"
+	item_state = "godeye"
+	lighting_alpha = LIGHTING_PLANE_ALPHA_MOSTLY_INVISIBLE
+	hud_type = DATA_HUD_SECURITY_BASIC
+
+/obj/item/clothing/glasses/hud/toggle/eldritch_eye/equipped(mob/living/carbon/human/user, slot)
+	. = ..()
+	if(ishuman(user) && slot == ITEM_SLOT_EYES)
+		ADD_TRAIT(src, TRAIT_NODROP, EYE_OF_GOD_TRAIT)
+
+/obj/item/clothing/glasses/hud/toggle/eldritch_eye/attack_self(mob/user)
+	..()
+	switch (hud_type)
+		if (DATA_HUD_MEDICAL_BASIC)
+			icon_state = "godeye"
+		if (DATA_HUD_SECURITY_BASIC)
+			icon_state = "godeye"
+		if (DATA_HUD_DIAGNOSTIC_BASIC)
+			icon_state = "godeye"
+		else
+			icon_state = "godeye"
+	user.update_inv_glasses()
+	
```

Comment:
this doesn't do much of anything just gives you sec hud

Reply:
Sorry if it wasn't clear, but when you initially equip it you gain the sechud, then pressing the hud button toggles between the HUD types.